packages/webview_flutter/ios/Classes/FlutterWebView.m

```diff
@@ -154,6 +179,33 @@ - (void)onEvaluateJavaScript:(FlutterMethodCall*)call result:(FlutterResult)resu
              }];
 }
 
+- (void)onAddJavaScriptChannels:(FlutterMethodCall*)call result:(FlutterResult)result {
+  NSArray* channelNames = [call arguments];
+  NSSet* channelNamesSet = [[NSSet alloc] initWithArray:channelNames];
+  [_javaScriptChannelNames addObjectsFromArray:channelNames];
+  [self registerJavaScriptChannels:channelNamesSet
```

Comment:
What happens if there is a duplicated value in channelNameSet with the global _javaScriptChannelNames which has been registered before? Is it safe to run the register again with the same channel?